docs/testcases/UI/Run_tools_in_sandbox/Launch_parameters/3122_insufficient_capacity/3298_2.md

```diff
@@ -0,0 +1,46 @@
+# Restart jobs in other regions should work if Allow reschedule option is enabled
+
+Test verifies that restart jobs in other regions should work if preference `run.reschedule.enabled` is set to disable but _Allow reschedule to different region in case of insufficient capacity_ option for tool is set to enable.
+
+**Prerequisites**:
+- Admin user
+- Perform [3122_1](3122_1.md) case
+- Node type `type1` of rare compute instance type for the current Cloud Provider which launch attempt can lead to the exception of insufficient capacity of such node types for the Cloud Region
+
+**Preparations**
+1. Login as admin user from the prerequisites
+2. Open the **Settings** page
+3. Click the **PREFERENCES** tab
+4. Find the **`run.reschedule.enabled`** preference
+5. Uncheck *Enabled* checkbox for `run.reschedule.enabled` preference if needed
+6. Click **Save** button if needed
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select test tool | |
+| 4 | Open **Settings** tab | |
+| 5 | Expand **EXECUTION ENVIRONMENT** | |
+| 6 | Select ***Enabled*** option for _Allow reschedule to different region in case of insufficient capacity_ | |
+| 7 | Click **Save** button if needed | |
+| 8 | At the tool page, hover over the **Run v** button | |
+| 9 | Click the **Custom settings** button in the list | |
+| 10 | Expand the **Exec environment** section | |
+| 11 | Select the **Node type** from the prerequisites | |
+| 12 | Select `region1` value (same as in step 4 of the preparations of case [3122_1](3122_1.md)) for **Cloud Region** | |
+| 13 | Expand the **Advanced** section | |
+| 14 | Select `On-demand` value for **Price type** | |
+| 15 | Launch the run | |
+| 16 | At the **Runs** page, click the just-launched run | |
+| 17 | Wait until ***RestartPipelineRun*** task appears | <li> Run is stopped <li> Message `Server failure reason: Insufficient instance capacity.` appears under the run title |
+| 18 | Wait until ***RestartPipelineRun*** task appears | <li> Run is stopped <li> Message `Server failure reason: Insufficient instance capacity.` appears under the run title <li> Message `The following run was launched instead of current one: <restarted_runID> <region2>` with link appears |
+| 19 | Click ***RestartPipelineRun*** task | Records `Run '<runID>' failed to start. <br> A new run '<restarted_runID>' started in a region '<region2>'.` are shown in the log |
+| 20 | Click on the `<restarted_runID>` in the `The following run was launched instead of current one` message | Restarted Run Log page opens that contains message `This run was launched instead of: <runID> <region1>` with link, where <runID> is ID of run launched at step 10 |
```

Comment:
Fix to: `... where <runID> is ID of run launched at step 15 |`

Reply:
Done